### DISCUSSION TITLE
API: Improve response of `DELETE /status/{id}`

### DISCUSSION THREAD

**Discussion by marhei** [Traewelling/traewelling#2060] [Ideas / Feature Requests]
Instead of a status element `success`, as specified in the docs, you get back a message element with a localised message in it

**Comment by marhei** [comment]
Also eigentlich ist das eher ein Bug und kein Feature Request: Abweichung API Realität und Dokumentation…

**Reply by HerrLevin** [reply]
Oh. Da ist wohl einer von uns Mausgerutscht beim Migrieren.

Ich schau morgen mal, ob ich das wieder zurück zu den Issues schieben kann. Ansonsten lege gerne auch das issue neu an und schließe die Diskussion. :)
